### PR TITLE
DOC-33773-2: new fixes

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by davidcurrie — APPROVED** [review]
LGTM!

File: content/en/platform/corda/4.8/open-source/tools-index.md [hunk 1]
```diff
@@ -24,6 +24,6 @@ wish to try the [Blob Inspector](blob-inspector.md).
 
 * [Corda Network Builder](network-builder.md)
 * [Network Bootstrapper](network-bootstrapper.md)
-* [DemoBench - **Warning** - DemoBench is deprecated in Corda 4.8](demobench.md)
+* [DemoBench - Warning: DemoBench is deprecated in Corda 4.8](demobench.md)
```
Comment (michelemarsden) [inline content/en/platform/corda/4.8/open-source/tools-index.md:27]:
```suggestion
```
I must have missed this one, as there was another ticket to remove DemoBench references in 4.7+

Comment (michelemarsden) [reply content/en/platform/corda/4.8/open-source/tools-index.md:27]: You might get a merge conflict as I had already deleted this line...

**Review by davidcurrie — APPROVED** [review]

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**